config/application.rb

```diff
@@ -110,8 +110,27 @@ class Application < Rails::Application
 
     config.autoload_paths += config.eager_load_paths
 
-    # config.load_defaults 6.1
-    # Disable defaults as ActiveRecord::Base.belongs_to_required_by_default = true causes MiqRegion.seed to fail validation on belongs_to maintenance zone
+    # FYI, this is where load_defaults is defined as of 7.2:
+    # https://github.com/rails/rails/blob/d437ae311f1b9dc40b442e40eb602e020cec4e49/railties/lib/rails/application/configuration.rb#L92
+    config.load_defaults 7.1
+
+    # TODO: this is the only change we had from defaults in 7.0.  See secure_headers.rb.  It's 0 in defaults.
+    config.action_dispatch.default_headers["X-XSS-Protection"] = "1; mode=block"
```

Comment:
Can we add a comment either here or in the secure_headers.rb that the reason we are keeping it is because we still use unsafe-inline, and once we can remove unsafe-inline then we can set this to the default.  Maybe also reference that paragraph in https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection

Reply:
Good idea.  Added in secure_headers.rb.  Pushed in 9c92e62242ac2392835f01759de465018a5ee58d, 👇